src/deployment/windows.md

```diff
@@ -108,6 +108,30 @@ so, Codemagic requires
 [associating the Azure Active Directory
 and Partner Center accounts][azureadassociation].
 
+#### Github Actions CI/CD
+
+Github Actions may use the
+[Microsoft Dev Store CLI](https://learn.microsoft.com/windows/apps/publish/msstore-dev-cli/overview)
+to package applications into an MSIX and publish them to the Microsoft Store.
+The [setup-msstore-cli](https://github.com/microsoft/setup-msstore-cli)
+Github Action installs the cli so that the Action can use it for packaging
+and publishing.
+
+As packaging the MSIX utilizes the
+[`msix` pub package][msix package], the project's `pubspec.yaml`
+must contain an appropriate `msix_config` node.
+
+The application must already be present in the Microsoft Dev Center with at
+least one complete submission, and `msstore init` must be run once within
+the repository before the Action can be performed. Once complete, running
+`msix package .` and `msix publish` in a Github Action will package the
+application into an MSIX and upload it to a new submission on the dev center.
+
+An example Action yml file for continous deployment may be found
```

Comment:
This example is long as it includes content that isn't relevant to publishing to the Microsoft Store. Should we include an excerpt of the YAML the user will need to add to their project? Something like:

```
      - uses: microsoft/setup-msstore-cli@v1

      - name: Configure the Microsoft Store CLI
        run: msstore reconfigure --tenantId ${{ secrets.AZURE_AD_TENANT_ID }} --clientId ${{ secrets.AZURE_AD_ClIENT_ID }} --clientSecret ${{ secrets.AZURE_AD_CLIENT_SECRET }} --sellerId ${{ secrets.SELLER_ID }}

      - name: Create MSIX
        run: msstore package .

      - name: Publish MSIX to the Microsoft Store
        run: msstore publish -v
```